packages/forms-web-app/src/config.js

```diff
@@ -76,7 +76,8 @@ module.exports = {
 		supportTeamPhoneNumber: '0303 444 5000',
 		pressAndMediaPhoneNumber: '0303 444 5004',
 		pressAndMediaEmailAddress: 'press.office@planninginspectorate.gov.uk',
-		contactUsFormURL: 'https://contact-us.planninginspectorate.gov.uk/hc/en-gb/requests/new'
+		contactUsFormURL: 'https://contact-us.planninginspectorate.gov.uk/hc/en-gb/requests/new',
+		openingHours: '9:00 to 17:00'
```

Comment:
if possible, try to be a bit more descriptive with variable names ( within reason ofc ;) ), so other devs have some context on what information is held by it. `openingHours` is a bit vague, `csOpeningHours` is only a bit longer but it we can immediately infer it relates to custommer service opening times.